### ISSUE TITLE
module: global module should be high priority than others

### ISSUE THREAD

**Issue by yorkie** [Rokid/ShadowNode#262]
* **Version**: v0.9.0
* **Platform**: all
* **Subsystem**: module

Currently, the module searching path are local module is higher than global, that's incorrect. For instance, take a look at the following:

```
/usr/local/lib/node_modules/foobar/index.js
/opt/my_projects/foobar/index.js
/opt/my_projects/test.js
```

In `/opt/my_projects/test.js`, `require('foobar')` loads the `/opt/my_projects/foobar/index.js`.

**Comment by legendecas** [comment]
Is this a path pattern issue? i.e. local packages must be loaded with relative path, `require('./foobar')`

**Comment by yorkie** [comment]
Yea, IoT.js support this without the relative path dot, for compatibility, we have to tweak the priority at the first step :)

**Comment by legendecas** [comment]
resolved with #274
